src/map/packets_struct.h

```diff
@@ -1423,21 +1423,21 @@ struct packet_party_leader_changed {
 	uint32 new_leader_aid;
 } __attribute__((packed));
 
-struct packet_hotkey {
 #ifdef HOTKEY_SAVING
+struct hotkey_data {
+	int8 isSkill; // 0: Item, 1:Skill
+	uint32 ID;    // Item/Skill ID
+	int16 count;  // Item Quantity/Skill Level
+};
```

Comment:
Shouldn't this also be `__attribute__((packed))`?

Reply:
yes